Resources/Prototypes/Entities/Structures/Specific/bluespace_harvster_rift.yml

```
@@ -0,0 +1,102 @@
+    mode: SnapgridCenter
+  components:
+  - type: BluespaceHarvesterRift
+    passiveSpawn:
+    - MobFleshJared
+    - MobFleshGolem
+    - MobFleshClamp
+    - MobFleshLover
+    - FleshBlocker
+    spawn:
+      # Flesh
+      - id: MobFleshJared
```

yamllint (1.35.1)

[warning] wrong indentation: expected 4 but found 6

(indentation)